helm/dashboards/charts/private_dashboards_mz/dashboards/shared/private/operatorkit.json

```diff
@@ -970,14 +970,14 @@
           "type": "prometheus",
           "uid": "${datasource}"
         },
-        "definition": "label_values(operatorkit_controller_operation_count{app=\"$app\"}, operation)",
```

Comment:
I'm not sure this works with old versions of chart/app-operator though

Reply:
I don't think it works in production either:
https://grafana.g8s.anteater.dub.aws.k8s.3stripes.net/d/piJK9Vm4z/operatorkit?orgId=1&refresh=5m

Reply:
Maybe don't post customer urls in public repos :D

Reply:
For this dashboard, by using `job` instead of `app`, we are getting sometimes `namespace/name`:
![image](https://github.com/giantswarm/dashboards/assets/9134338/86d68223-0c8f-4847-a57f-484cf04585c9)
vs
![image](https://github.com/giantswarm/dashboards/assets/9134338/0323a844-be8e-4b24-9af0-7b6b6edfe35b)

Reply:
For old releases without ServiceMonitor, we will get something like:
`dc746-prometheus/workload-dc746/0` 
IMHO it's temporary and acceptable, wdyt ?

Reply:
The one with namespace is for pormonitors but sure